set.mm

```diff
@@ -345171,6 +345369,224 @@ The second section ("Tarskian geometry") develops the synthetic treatment of
+          KYLYMYICDFBGHXBJXCKYJYLYKYNXNWIUGUOYIECFBGHXBJXCKYJYMYLYNWEWBWIUKUOUP
+          VKVLAXLWDWBXMUFUTVCVDAWCUMWDWAAWDWCTUDAWDWCWAWEWCUMZWGWAWHWIYOWGUMZBG
+          HIDECJLKWEWKWCWGWLUFZWEWQWCWGWRUFZWEWDWCWGWTUFZWEWNWCWGWOUFZYPCFDEBGH
+          IJKYQYTWEXDWCWGXEUFYRYSLAXTWDWCWGYAVFYOWGUGWEWCWGUKVMVJYOWHUMZBGHIDEC
+          JLKWEWKWCWHWLUFZWEWQWCWHWRUFZWEWDWCWHWTUFZWEWNWCWHWOUFZUUABGHIDECJLKU
+          UBUUCUUDUUEUUAECDBGHXBJXCKUUBUUDUUEUUCUUAEFCDBGHXBJXCKUUBUUDWEXDWCWHX
+          EUFZUUEUUCAYHWDWCWHRVFUUACFEBGHXBJXCKUUBUUEUUFUUDWEWCWHUKUOYOWHUGVNUO
+          UNVJYOWIUMZBGHICDEJLKWEWKWCWIWLUFZWEWNWCWIWOUFZWEWQWCWIWRUFZWEWDWCWIW
+          TUFZUUGCDFEBGHXBJXCKUUHUUIUUJWEXDWCWIXEUFUUKYOWIUGWEWCWIUKULVKAXLWDWC
+          XMUFUTVCVDAAWDECFIUBUCZVSWBWCURZAVQTUAWEUULUUMWEBGHICFEJLKWLWOXEWTUSV
+          OVPUT $.
+      $}
+
+      $d x L $.  $d x P $.  $d x Q $.  $d x S $.  $d x ph $.
+      $( If ` S ` lies on PQ , then PQ = PS .  Theorem 6.16 of [Schwabhauser]
+         p. 45.  (Contributed by Thierry Arnoux, 17-May-2019.) $)
+      tglineelsb2 $p |- ( ph -> ( P L Q ) = ( P L S ) ) $=
+        ( wcel syl vx co cv cstrkg simpl wne necomd lncom lnrot1 tglnssp sselda
+        wa simpr tglineeltr impbida eqrdv ) AUACDHUBZCEHUBZAUAUCZUQSZUSURSZAUTU
+        LZBCEUSDFGHIJKVBAFUDSZAUTUEZLTZVBACBSZVDMTZVBAEBSZVDPTZVBACEUFVDAECQUGT
+        VBADBSZVDNTZVBADCUFVDACDOUGTVBBFGHDCEIKJVEVKVGVIVBBFGHCDEIKJVEVGVKVIVBA
+        EUQSZVDRTUHUIAUQBUSABFGHCDIKJLMNUJUKAUTUMUNAVAULZBCDUSEFGHIJKVMAVCAVAUE
+        ZLTVMAVFVNMTVMAVJVNNTVMACDUFVNOTVMAVHVNPTVMAECUFVNQTVMAVLVNRTAURBUSABFG
+        HCEIKJLMPUJUKAVAUMUNUOUP $.
+    $}
+
+    ${
+      $( Reflexivity law for line membership.  Part of theorem 6.17 of
+         [Schwabhauser] p. 45.  (Contributed by Thierry Arnoux,
+         17-May-2019.) $)
+      tglinerflx1 $p |- ( ph -> P e. ( P L Q ) ) $=
+        ( cds cfv eqid tgbtwntriv1 btwnlng1 ) ABEFGCDCHJIKLMLACDBEFENOZHSPIKLMQ
+        R $.
+
+      $( Reflexivity law for line membership.  Part of theorem 6.17 of
+         [Schwabhauser] p. 45.  (Contributed by Thierry Arnoux,
+         17-May-2019.) $)
+      tglinerflx2 $p |- ( ph -> Q e. ( P L Q ) ) $=
+        ( cds cfv eqid tgbtwntriv2 btwnlng1 ) ABEFGCDDHJIKLMMACDBEFENOZHSPIKLMQ
+        R $.
+
+      $d x L $.  $d x P $.  $d x Q $.  $d x ph $.
+      $( Commutativity law for lines.  Part of theorem 6.17 of [Schwabhauser]
+         p. 45. (Contributed by Thierry Arnoux, 17-May-2019.) $)
+      tglinecom $p |- ( ph -> ( P L Q ) = ( Q L P ) ) $=
+        ( vx co wcel wa adantr tglnssp sselda cstrkg simpr lncom impbida eqrdv
+        cv ) ANCDGOZDCGOZANUFZUGPZUIUHPZAUJQBEFGCDUIHJIAEUAPZUJKRACBPZUJLRADBPZ
+        UJMRAUGBUIABEFGCDHJIKLMSTAUJUBUCAUKQBEFGDCUIHJIAULUKKRAUNUKMRAUMUKLRAUH
+        BUIABEFGDCHJIKMLSTAUKUBUCUDUE $.
+
+      $d A x y $.  $d B x y $.  $d G y x $. $d I x y $.  $d L x y $.
+      $d P x y $.  $d Q x y $.  $d ph x y $.
```

Comment:
I love that you already have a nontrivial amount in chapter 6.

Reply:
Scott Fenton went up to 6.21, so with 6.18, I've already almost covered what he had, but what I really want to cover is the chapter 14... that's still a long way to go.